src/components/DraggableViewSection.tsx

```diff
@@ -14,17 +21,34 @@ export interface DraggableViewSectionProps {
   onDimensionChange: (change: number) => void;
 }
 
-function DraggableViewSection({ children, offset, dimension, vertical, dimensionsComputed, isLast, onDimensionChange }: DraggableViewSectionProps) {
+let dragEvent: DragEvent<Body> | null = null;
```

Comment:
Lets make this a [ref](https://react.dev/learn/referencing-values-with-refs) instead of using a global variable here. This component could be used multiple times on the page.